drivers/include/dose.h

```diff
@@ -175,8 +175,10 @@ typedef struct {
  * @brief   Setup a DOSE based device state
  * @param[out]  dev         Handle of the device to initialize
  * @param[in]   params      Parameters for device initialization
+ * @param[in]   index       index of @p params in a global parameter struct array.
```

Comment:
Here the capitalization of the parameter documentation is different.

```suggestion
 * @param[in]   index       Index of @p params in a global parameter struct array.
```